routes/hot_feed.go

```diff
@@ -764,6 +1016,17 @@ func (fes *APIServer) AdminUpdateHotFeedAlgorithm(ww http.ResponseWriter, req *h
 		}
 	}
 
+	if requestData.InteractionCapTag > 0 {
+		err := fes.GlobalState.Put(
+			_GlobalStatePrefixForHotFeedTagInteractionCap,
+			lib.EncodeUint64(uint64(requestData.InteractionCapTag)),
+		)
+		if err != nil {
+			_AddInternalServerError(ww, fmt.Sprintf("AdminUpdateHotFeedAlgorithm: Error putting InteractionCapTag: %v", err))
+			return
+		}
+	}
```

Comment:
probably worth refactoring - define a function to store a constant in global state